packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -14,6 +14,55 @@ import type { ThemeOptions } from "@mui/material";
 //import radioClasses from "@mui/material";
 
 export const components: ThemeOptions["components"] = {
+  MuiAlert: {
+    styleOverrides: {
+      root: ({ ownerState, theme }) => ({
+        padding: theme.spacing(4),
+        color: theme.palette.text.primary,
+        ...(ownerState.severity && {
+          backgroundColor: theme.palette[ownerState.severity].lighter,
+          borderColor: theme.palette[ownerState.severity].light,
+        }),
+        ...(ownerState.variant === "infobox" && {
+          borderStyle: "solid",
+          borderWidth: 1,
+          "&:not(:last-child)": {
+            marginBottom: theme.spacing(4),
+          },
+        }),
+        ...(ownerState.variant === "banner" && {
+          borderWidth: 0,
+        }),
+      }),
+      icon: ({ ownerState, theme }) => ({
+        marginRight: theme.spacing(4),
+        padding: 0,
+        fontSize: "1.429rem",
```

Comment:
We'll need to create custom variables to store tokens like `FontScaleX` when they aren't associated with a Heading style.